roles/ceph-common/defaults/main.yml

```diff
@@ -302,6 +306,15 @@ radosgw_keystone_token_cache_size: 10000
 radosgw_keystone_revocation_internal: 900
 radosgw_s3_auth_use_keystone: "true"
 radosgw_nss_db_path: /var/lib/ceph/radosgw/ceph-radosgw.{{ ansible_hostname }}/nss
+# Settings for the RGW usage logging described at http://docs.ceph.com/docs/jewel/man/8/radosgw/#usage-logging
+radosgw_usage_log: false
+radosgw_usage_log_tick_interval: 30
```

Comment:
@Logan2211 yeah, I do see that in the template. I just asked because ``ceph_conf_overrides`` apparently supports adding or overriding an ``rgw`` section. https://github.com/ceph/ceph-ansible/blob/master/roles/ceph-common/defaults/main.yml#L326